cmd/kubeadm/app/cmd/phases/workflow/runner.go

```diff
@@ -330,9 +330,14 @@ func (e *Runner) BindToCommand(cmd *cobra.Command) {
 	e.prepareForExecution()
 
 	// adds the phases subcommand
+	const phaseCommandUse = "phase"
 	phaseCommand := &cobra.Command{
-		Use:   "phase",
+		Use:   phaseCommandUse,
 		Short: fmt.Sprintf("Use this command to invoke single phase of the %s workflow", cmd.Name()),
+		Args:  cobra.NoArgs,
```

Comment:
this tells the cmd that anything after e.g. `kubeadm init phase ...` is not an arg but a subcommand.